src/Maestro/Maestro.Data/BuildAssetRegistryContext.cs

```diff
@@ -39,10 +38,16 @@ public BuildAssetRegistryContext CreateDbContext(string[] args)
                 opts.CommandTimeout(30 * 60);
             })
             .Options;
+
         return new BuildAssetRegistryContext(
             new HostingEnvironment{EnvironmentName = Environments.Development},
             options);
     }
+
+    public static string GetConnectionString(string databaseName)
+    {
+        return $@"Data Source=localhost\SQLEXPRESS;Initial Catalog={databaseName};Integrated Security=true"; // CodeQL [SM03452] This 'connection string' is only for the local SQLExpress instance and has no credentials, Encrypt=true is unnecessary
```

Comment:
This `CodeQL [SM03452]` comment will suppress the warning?

Reply:
Ah no, that's just the issue number or something. There might be a way to suppress it completely tho

Reply:
I know I had to add this https://github.com/dotnet/dnceng/commit/0290444491dfc3a31da0be480e59f611224dba0c to suppress a different warning, there might be something similar for this case

Reply:
In this PR, there were 3 places where this was used. One had the `// lgtm`, one had the `// CodeQL [SM03452]` and one had nothing. I went with CodeQL because it matches the linked issue that we have opened - the number matches the problem ID too.